doc/content/modules/user-manual/pages/release-notes/2025.2.0.adoc

```diff
@@ -103,6 +103,11 @@ image::release-notes-diagrams-export-as-png.png[Export diagram as PNG, width=50%
 
 - `Membership#isDistinguishableFrom` derived attribute has been implemented in the metamodel.
 
+- SysML textual export now correctly exports the following concepts :
```

Comment:
Please fix the sentence. Something like `The following concepts are now handled by the export to textual SysMLv2 module:...`

Reply:
Done.